students/byeonghwi/users/views.py

```diff
@@ -36,7 +36,28 @@ def post(self, request):
         phone    = phone,
       )
 
-      return JsonResponse({'message': 'success'}, status = 201)
+      return JsonResponse({'message': 'SUCCESS'}, status = 201)
+
+    except KeyError:
+      return JsonResponse({'message': 'KEY_ERROR'}, status = 400)
+
+
+class LoginView(View):
+
+  def post(self, request):
+
+    try:
+      data     = json.loads(request.body)
+      email    = data['email']
+      password = data['password']
+      
+      if not User.objects.filter(email = email).exists():
+        return JsonResponse({'message':'INVALED_USER'}, status = 401)
+
+      if not User.objects.filter(password = password).exists():
+        return JsonResponse({'message':'INCORRECT_PASSWORD'}, status = 401)
```

Comment:
아래 코드는 버그를 발생할 수 있는 코드입니다. password가 동일한 유저는 얼마든지 있을 수 있기 때문입니다.
그렇기 때문에 email과 password를 같이 조회해야 원하는 결과를 얻을 수 있습니다.

```suggestion
      if not User.objects.filter(email = email, password = password).exists():
        return JsonResponse({'message':'INCORRECT_PASSWORD'}, status = 401)
```

Reply:
@lordmyshepherd 감사합니다 수정완료했습니다.